build/yaml/steps/microbuild/codesignverify.yaml

```diff
@@ -0,0 +1,20 @@
+# Step Template: MicroBuild Signing and Verification
+
+parameters:
+    SignType: None
+
+steps:
+
+- ${{ if notIn(parameters.SignType, '', 'None') }}:
```

Comment:
>if notIn(parameters.SignType, '', 'None') [](start = 6, length = 41)

Consider doing the positive approach, if the parameter is set to "Real" or "Test", then set the variable.

Reply:
I wasn't able to get it working earlier, I'll try this again

---
In reply to: [356695467](https://github.com/microsoft/CLRInstrumentationEngine/pull/181#discussion_r356695467) [](ancestors = 356695467)

Reply:
Turns out we're pulling in a build variable which happens after the yaml generation.

---
In reply to: [356724316](https://github.com/microsoft/CLRInstrumentationEngine/pull/181#discussion_r356724316) [](ancestors = 356724316,356695467)